.github/workflows/ci.yml

```
@@ -18,48 +18,7 @@ jobs:
           version: 10
       - uses: actions/setup-node@v4
         with:
-          version: 'lts/*'
-          cache: 'pnpm'
+          version: "lts/*"
```

actionlint (1.7.7)

input "version" is not defined in action "actions/setup-node@v4". available inputs are "always-auth", "architecture", "cache", "cache-dependency-path", "check-latest", "node-version", "node-version-file", "registry-url", "scope", "token"

(action)